Apps/Sandcastle/gallery/Imagery Layers Split.html

```diff
@@ -68,27 +68,41 @@
 var slider = document.getElementById('slider');
```

Comment:
Not really relevant to your changes, but can you add `infoBox: false` as an option when creating the viewer? It will keep the infobox from popping up without any information.